build/main/java/org/greencodeinitiative/tools/exporter/Main.java

```diff
@@ -0,0 +1,66 @@
+package org.greencodeinitiative.tools.exporter;
+
+import org.greencodeinitiative.tools.exporter.infra.MetadataWriter;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+import static java.util.Optional.empty;
+import static java.util.Optional.ofNullable;
+
+public class Main implements Runnable {
+    private final List<String> args;
+
+    public Main(String[] args) {
+        this.args = ofNullable(args).map(List::of).orElse(emptyList());
+    }
+
+    public static void main(String... args) {
+        new Main(args).run();
+    }
+
+    @Override
+    public void run() {
+        Path sourceDir = argAsPath(0, "sourceDir");
+        Path targetDir = argAsPath(1, "targetDir");
+        new MetadataWriter(
+                sourceDir,
+                targetDir,
+                Map.of(
+                        "title", arg(2, "specificationTitle"),
+                        "version", arg(3, "specificationVersion"),
+                        "scmRevisionNumber", arg(4, "specificationSCMRevisionNumber"),
+                        "scmRevisionDate", arg(5, "specificationSCMRevisionDate")
+                ),
+                // indexFile
+                optionalArg(6)
+                        .map(Path::of)
+                        .orElseGet(() -> targetDir.resolve("index.json")),
```

Comment:
Do we really need the `index.json` inside the JAR files?
I say this because the website won't be able to use it, it needs a URL with direct access to the json file

Reply:
Indeed, the multilingual JAR file is not currently needed.

Therefore, there is no need to create and publish this specific JAR file.

On the other hand, creating an "index.json" file is a good idea to expose the rules as an API via GitHub Pages.